front-end/app/src/pages/login/Form.js

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import axios from "axios";
+import Navbar from "../../components/nav/Nav";
+import HomePage from "./../home/Home";
+import BookStandPage from "./../book-stand/BookStand";
+import AboutPage from "./../about/About";
+import SettingsPage from "./../settings/Settings";
+import {BrowserRouter as Router, Switch, Route } from "react-router-dom";
+export default class Form extends React.Component {
+
+  state = {
```

Comment:
Interesting use of state, but please confirm with Audrey whether it's ok to not use a constructor to set up the state of your class, and whether your setState functions will still work.